content/en/opentelemetry/instrument/api_support/dotnet/metrics.md

```diff
@@ -0,0 +1,92 @@
+---
+title: .NET OpenTelemetry Metrics API Support
+code_lang: metrics
+type: multi-code-lang
+code_lang_weight: 1
+further_reading:
+    - link: opentelemetry/correlate/metrics_and_traces
+      tag: Documentation
+      text: Correlate OpenTelemetry Traces and Metrics
+---
+
+## Overview
+
+{{% otel-overview-native lang=".NET" signal="Metrics" sdk_name="dd-trace-dotnet" %}}
+
+## Prerequisites
+
+- **Datadog SDK**: dd-trace-dotnet version [x.y.z] or later.
```

Comment:
Our support only runs on applications running on .NET 6 or newer. Maybe we add a line something like this?
```suggestion
- The instrumented application is running on .NET 6 or later.
- **Datadog SDK**: dd-trace-dotnet version [x.y.z] or later.
```